config/config/sc-config.yaml

```diff
@@ -1041,3 +1041,10 @@ welcomingDashboard:
   extraTextGrafana: ""
   # For newline in Opensearch dashboard use format "\\\\n"
   extraTextOpensearch: ""
+
+network:
+  defaultDeny: false
+  allowKubeSystem: true
+  allowDns: true
+  additionalEgressPolicies: []
+  additionalIngressPolicies: []
```

Comment:
You could move this and the config in wc-config.yaml to common-config if you want.

Also, would it make sense to be able to allow more namespaces via values?

Reply:
Even if we put this in common-config you can still override that for a single cluster, e.g. for sc by adding the part you want to change in your own sc-config. Moving this to common-config just shows that this is config that is available for both and that the defaults are the same.

I was thinking of a list where the one can add whatever namespace one wants.

Reply:
Could you also add a comment saying that this requires one to have calico as the network plugin?

Reply:
I moved it to common-config, added a variable for a list of namespaces, and added a comment.